config-example.sh

```diff
@@ -89,15 +89,17 @@ export NOCOL='\033[00m'
 
 # write log entries in debugfile - single line as second argument
 debug() {
-  line="$*"
-  echo "[$(date '+%H:%M:%S')] $line" >> $DEBUGFILE;
+  local ___line
```

Comment:
why so many `_`? Is that a new convention I'm not aware of?

Reply:
These are _rage-scores_: every underscore is equivalent to roughly 1h of debugging ;)
```
$ debug <<< $line
foo
$ debug <<< $line

```